tests/page-objects/home-page.ts

```diff
@@ -1,18 +1,19 @@
 import { type Page, Locator, expect } from '@playwright/test';
-import { SearchPage } from './search-page';
+
+import { CollectionBrowser } from './collection-browser';
 
 export class HomePage {
   readonly page: Page;
 
   readonly waybackSearch: Locator;
 
-  readonly searchPage: SearchPage;
+  readonly collectionBrowser: CollectionBrowser;
 
   public constructor(page: Page) {
     this.page = page;
     this.waybackSearch = this.page.locator('ia-wayback-search');
 
-    this.searchPage = new SearchPage(page);
+    this.collectionBrowser = new CollectionBrowser(page);
   }
```

Comment:
Just for clarification, the `CollectionBrowser` type in these tests is not the component itself, but rather a helper class for _interacting_ with the component, correct?

Not a blocker for this PR, but in future I wonder whether it would make sense to adjust the naming of these helper classes to avoid confusion.